ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs

```diff
@@ -32,7 +32,7 @@ use tonic::Status;
 type EndVersion = u64;
 
 const SERVICE_TYPE: &str = "indexer_fullnode";
-const MINIMUM_TASK_LOAD_SIZE_IN_BYTES: usize = 1_000_000;
+const MINIMUM_TASK_LOAD_SIZE_IN_BYTES: usize = 100_000;
```

Comment:
What would be the typical number of p2p transactions processed per task after this change?

Reply:
It also depends on how other parameter is set. In one of the test I'm seeing 3200 txns can be processed in ~60ms for this decoding step, so around 50K for this part.